runtimes/standalone/src/lib.rs

```diff
@@ -380,9 +379,9 @@ parameter_types! {
 
 impl did::Config for Runtime {
 	type DidIdentifier = DidIdentifier;
-	type Event = Event;
-	type Call = Call;
-	type Origin = Origin;
+	type RuntimeEvent = RuntimeEvent;
+	type Call = RuntimeCall;
```

Comment:
The sudo pallet uses `RuntimeCall`, so I guess we should use the same here.

Reply:
fde113cb1b81146770db9d3edfab89e40100617a